t/node/http_host.t

```diff
@@ -64,5 +64,5 @@ GET /uri
 Host: foo.com:1984
 --- response_body
 uri: /uri
-host: foo.com:1984
+host: foo.com
```

Comment:
same issue, I do not know why we need to update this test case

Reply:
I change `$http_host` to `$host`, ref https://github.com/apache/apisix/pull/10989/files#r1510533390
`$host` has no `port`. So I will lost the port info here.